frameworks/CSharp/aspnetcore/Benchmarks/Controllers/FortunesController.cs

```diff
@@ -11,6 +11,16 @@ namespace Benchmarks.Controllers
     [Route("mvc/fortunes")]
     public class FortunesController : Controller
     {
+        private readonly RavenDb _db;
+
+        public FortunesController()
+        { }
+
+        public FortunesController(RavenDb provider)
```

Comment:
Shouldn't this be used via `HttpContext.RequestServices.GetRequiredService`, like the others?

Reply:
Not really, unless we want to introduce a scope factory.